addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection.xml

```diff
@@ -4,11 +4,9 @@
     <t t-name="mail.DiscussMobileMailboxSelection" owl="1">
         <div class="o_DiscussMobileMailboxSelection">
             <t t-foreach="orderedMailboxes" t-as="mailbox" t-key="mailbox.localId">
-                <button class="o_DiscussMobileMailboxSelection_button btn"
+                <button class="o_DiscussMobileMailboxSelection_button btn btn-secondary"
                     t-att-class="{
-                        'btn-primary': discuss.thread === mailbox,
-                        'btn-secondary': discuss.thread !== mailbox,
-                        'o-active': discuss.thread === mailbox,
+                        'active o-active': discuss.thread === mailbox,
```

Comment:
cool for keeping `o-active`. Note that when all UI things will eventually be integrated to Models, the `o-active` would simply be a computed field... We could even just remove this as it's pretty much just an alias for `discuss.thread === mailbox`.

Reason to define it is because component logic and styling require introducing this classname, but it would no longer be necessary when all is model ;)